extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -3,15 +3,27 @@ define('AuthModal', ['jquery', 'AuthComponent', 'wikia.window'], function ($, Au
 
 	var modal,
 		$blackout,
-		language = window.wgContentLanguage;
+		modalEnabled,
+		language = window.wgContentLanguage,
+		track;
 
 	function open () {
+		if (modalEnabled) {
+			close();
+		}
 		$('.WikiaSiteWrapper').append(
-			'<div class="auth-blackout blackout visible"><div class="auth-modal loading"><a class="close"></div></div>'
+			'<div class="auth-blackout blackout visible"><div class="auth-modal loading"><a class="close" href="#"></div></div>'
 		);
+		modalEnabled = true;
 		$blackout = $('.auth-blackout');
 		modal = $blackout.find('.auth-modal')[0];
 		$('.auth-blackout, .auth-modal .close').click(close);
+
+		track = Wikia.Tracker.buildTrackingFunction({
```

Comment:
It's strange that we build tracker in ```open``` method and use it only in ```close```.

BTW what do you think about adding tracking for modal open event?